### ISSUE TITLE
Status bar is jumping after close cc

### ISSUE THREAD

**Issue by Vipzerohero** [midnightchip/HomeGesture-Full#61]
As description. When open cc by swipe right status bar then close cc by swipe cc up. The right status bar jump up alittle then get back as normal.
Im using ios 12.0.1 on iphone 6s plus and jailbream by unc0ver

**Comment by DT2808** [comment]
Yeah same here. When I exit the control centre, the status bar temporarily looks like the iPad one but then reverts back to your chosen status bar. The status bar is also a bit off so it looks jumpy. Screen record: https://imgur.com/a/KkqAKyi
